backend/controllers/game-controllers.js

```diff
@@ -217,6 +226,44 @@ const startRound = (io, roomId) => {
         roundPlayerRankings: playerRankings,
       });
 
+      // Calculate player multipliers based on points
```

Comment:
all this new code gives players the power up and updates the player objects. let's move it before the `roundEnded` event so we send the most up-to-date player objects to the client.